src/components/Loading/Loading.module.css

```diff
@@ -0,0 +1,37 @@
+
+div .loading-container {
+    width: 100%;
+    height: 100%;
+    background-color: rgba(#E4E4ED, 0.5);
+}
+
+div .bar {
+    width: 20rem;
+    height: 100%;
+    /* background-color: teal;; */
+    background: var(--blue-gradient);
+    border-radius: 0.5rem;
+    position: relative;
+    left: -15%;
+    animation: barAnimation 1.5s infinite ease-in-out;
+}
+
+@keyframes barAnimation {
+    0% {
+        left: -15%;
```

Comment:
Using left, right, top, bottom animations is not recommended as it is more computationally expensive . Is there a way of doing it with translate?